app_allocator/tests/test_event.py

```diff
@@ -0,0 +1,34 @@
+from app_allocator.classes.event import Event
+from app_allocator.tests.utils import assert_only_these_fields_in_csv_row
+
+EVENT_FIELDS = {'a': 'bcdef', 'b': 'cdefghi'}
+VALUE = 'tuvwxy'
+
+
+class TestEvent(object):
+    def test_event_to_csv(self):
+        event = Event(**EVENT_FIELDS)
+        csv_fields = event.to_csv().split(",")
+        assert all([field in csv_fields for field in EVENT_FIELDS.values()])
+
+    def update_event_helper(self, field):
+        event = Event(**EVENT_FIELDS)
+        event.update(**{field: VALUE})
+        assert event.fields[field] == VALUE
+
+    def test_update_event_existing_field(self):
+        self.update_event_helper('b')
```

Comment:
'b' and in the next function 'c' seem a bit magical.  Would be better if they had logical names like, EXISTING_FIELD and NEW_FIELD.